Cargo.toml

```diff
@@ -7,14 +7,15 @@ repository = "https://github.com/aurora-is-near/aurora-eth-connector"
 license = "CC0 1.0"
 
 [workspace.dependencies]
-aurora-engine-types = { git = "https://github.com/aurora-is-near/aurora-engine.git", branch = "develop", default-features = false }
+aurora-engine-types = { git = "https://github.com/aurora-is-near/aurora-engine.git", tag = "3.3.1", default-features = false }
```

Comment:
Why not `v3.5.0`?

Reply:
To be consistent with aurora-workspace: https://github.com/aurora-is-near/aurora-workspace/blob/main/Cargo.toml#L24

Reply:
Ok then, released:
https://github.com/aurora-is-near/aurora-workspace/releases/tag/0.4.1

Please update `aurora-engine` version and `aurora-workspace`.

Reply:
Could you please also release the migration tools with the deps fixes? So we can relplase the rev commit with tag

Reply:
Yes, sure.

Reply:
https://github.com/aurora-is-near/aurora-engine-migration-tool/pull/20

Reply:
721ac9201beb6fe3baadec1066b5882908c1dc4e